### DISCUSSION TITLE
Home screen is confused

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44152] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

It looks like https://github.com/ got a refresh:

<img width="861" alt="image" src="https://user-images.githubusercontent.com/2119212/212184525-e853e616-3e4f-4b62-bf96-36d8fd433623.png">

The highlighted text reads: `You're seeing this because you haven't used repositories, issues, and pull requests recently.`

I don't think any of that text is correct:

https://github.com/jsoref/
<img width="760" alt="image" src="https://user-images.githubusercontent.com/2119212/212184666-125573f2-bbe7-4961-9197-5173b7781852.png">

**Comment by brandonrosage** [comment, marked as answer]
Thanks, @jsoref. This was absolutely in error, and the Growth team was able to diagnose it. In fact, this content is currently not displayed to anyone, while our engineers work to ensure this mistake doesn't happen again. 

In short, most of this content is intended to display only to users who haven't used repos, issues, or pull requests in the past 28 days (and/or haven't previously dismissed this content).
